config/confighttp/confighttp.go

```diff
@@ -238,7 +246,14 @@ type HTTPServerSettings struct {
 
 // ToListener creates a net.Listener.
 func (hss *HTTPServerSettings) ToListener() (net.Listener, error) {
-	listener, err := net.Listen("tcp", hss.Endpoint)
+	network := hss.Network
+	if network == "" {
+		network = "tcp"
+	}
+	if network == "http+unix" || network == "https+unix" || network == "unixpacket" {
```

Comment:
is this a standard? if yes, can you link to a doc?

Reply:
No, I looked around and I didn't see a good standard scheme for HTTP over unix. I saw a number of approaches like this one. I would solicit anyone with expertise to help here.

Reply:
Then why supporting these values? Can we start only with "unix"?

Reply:
sure